### PR TITLE
created-messages-and-modal-for-them

### PR DESCRIPTION
**Pull request by Kuz1302**

### CODE REVIEW

File: frontend/screen/src/components/Footer/Messages/Message/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  const {link, text, openDialog} = props
+  const shortify = (text, maxlength = 100) => {
+    if (text.length < maxlength) {
+      return text
+    }
+    return `${text.slice(0, maxlength - 3)}...`
+  }
+  return (
+    <li onClick={openDialog} className={'messages__item'}>
+      {/* <img src={image} className={'messages__item__photo'} /> */}
```
Comment (javasia) [inline frontend/screen/src/components/Footer/Messages/Message/index.js]: no comments into master

File: frontend/screen/src/components/Footer/Messages/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          this.state.activeMessage &&
+          <Modal
+            isOpen={this.state.activeMessage}
+            contentLabel="onRequestClose Example"
+            onRequestClose={this.handleClose}
+            style={customStyles}
+            shouldCloseOnOverlayClick={true}
+          >
+            <div className={'messages__item'}>
+              {/* <img src={activeMessage.image} className={'messages__item__photo'} /> */}
```
Comment (javasia) [inline frontend/screen/src/components/Footer/Messages/index.js]: comments

File: frontend/screen/src/components/Footer/Messages/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
```
Comment (javasia) [inline frontend/screen/src/components/Footer/Messages/index.js]: Please add props validation

File: frontend/screen/src/components/Footer/Messages/Message/index.js (hunk 1 above)
Comment (Kuz1302) [reply frontend/screen/src/components/Footer/Messages/Message/index.js]: Done

File: frontend/screen/src/components/Footer/Messages/index.js (hunk 2 above)
Comment (Kuz1302) [reply frontend/screen/src/components/Footer/Messages/index.js]: Done

File: frontend/screen/src/components/Footer/Messages/index.js (hunk 3 above)
Comment (Kuz1302) [reply frontend/screen/src/components/Footer/Messages/index.js]: For what?

**Comment by Kuz1302** [comment]
[4SP Create component for Place messages (Bottom part of the screen)](https://trello.com/c/QFtNPIw1/161-4sp-create-component-for-place-messages-bottom-part-of-the-screen)

**Pull request merged**